.github/workflows/test.yaml

```diff
@@ -56,9 +56,9 @@ jobs:
           #   we test integration with. We can test against only one version and
           #   that would be fine.
           #
-          - python-version: "3.10"
+          - python-version: "3.12"
             go-version: "1.20"
-          - python-version: "3.11"
+          - python-version: "3.13"
```

Comment:
Let's keep testing python 3.10-3.13 here instead of down to 3.12 to 3.13

Reply:
Done :)